manage_nf.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
I might have missed it. When and where manage_nf.sh is called? Is this a test script?